scripts/protobuf_codegen.sh

```diff
@@ -9,7 +9,7 @@ fi
 # any version changes here should also be bumped in Dockerfile.buf
 # ref. https://docs.buf.build/installation
 # ref. https://github.com/bufbuild/buf/releases
-BUF_VERSION='1.23.1'
+BUF_VERSION='1.25.0'
```

Comment:
can we bump up version in Dockerfile.buf and others.

ref. https://github.com/ava-labs/avalanchego/commit/b60d7c3254ddd6bfe4de52cd6b3a10da37be4ec2